socorro/unittest/processor/test_signature_utilities.py

```diff
@@ -1117,15 +1076,15 @@ def test_action_2(self):
         }
 
         # the call to be tested
-        ok_(sgr._action(raw_crash, {}, processed_crash, processor_meta))
+        assert sgr._action(raw_crash, {}, processed_crash, processor_meta) is True
 
-        eq_(
-            processed_crash['signature'],
+        assert (
+            processed_crash['signature'] ==
```

Comment:
This seems too subtle. I think it's easy to lose the comparison operator. Above (line `1005` post diff) we avoid direct comparisons to multiline literals. Consider similar here and below.